applications/h264/h264_endoscopy_tool_tracking/cpp/metadata.json

```diff
@@ -0,0 +1,49 @@
+{
+  "application": {
+    "name": "H264 Endoscopy Tool Tracking",
+    "authors": [
+      {
+        "name": "Holoscan Team",
+        "affiliation": "NVIDIA"
+      }
+    ],
+    "language": "C++",
+    "version": "2.1",
+    "changelog": {
+      "1.0": "Initial Release",
+      "2.0": "Upgrade to GXF 4.0",
+      "2.1": "Import GXF Codelets/Components as Holoscan Operators/Resources"
```

Comment:
Change to `Import h.264 GXF Codelets/Components as Holoscan Operators/Resources`